docs/can-guides/commitment/recipes/credit-card-simple/2-read-form.js

```diff
@@ -1,17 +1,34 @@
+import { Component } from "can";
+
 Stripe.setPublishableKey("pk_test_zCC2JrO3KSMeh7BB5x9OUe2U");
 
-const PaymentVM = can.DefineMap.extend({
-  amount: { default: 9.99 },
+Component.extend({
+	tag: 'cc-payment',
+	view: `
+	<form>
 
-  userCardNumber: "string",
+		<input type="text" name="number" placeholder="Card Number
```

Comment:
The `placeholder` attribute is missing an end quote (here and other places).

I caught this because the syntax highlighting was off in the rendered page.

Reply:
Done!